optimade/filtertransformers/elasticsearch.py

```diff
@@ -286,3 +291,7 @@ def float_literal(self, args):
 
     def string_literal(self, args):
         return args[0].strip('"')
+
+
+# added for potential backward compatibility
+Transformer = ElasticTransformer
```

Comment:
We should add some deprecation warning here as well if this is imported.

Reply:
My plan was just to remove this alias in #613 now that Markus has seen it (presumably he is the only user at the moment...)

Reply:
Right. And he was okay with this from comments on Slack, correct?

Reply:
I believe so! It had already been renamed in his original PR too, so think we're fine.

Reply:
Right. And to keep his commit being the one that changes this is better than us doing it here. I get it 👍